core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -217,6 +217,7 @@ class KafkaApis(val requestChannel: RequestChannel,
         case ApiKeys.ENVELOPE => handleEnvelope(request)
         case ApiKeys.DESCRIBE_CLUSTER => handleDescribeCluster(request)
         case ApiKeys.DESCRIBE_PRODUCERS => handleDescribeProducersRequest(request)
+        case ApiKeys.UNREGISTER_BROKER => maybeForwardToController(request, handleUnregisterBrokerRequest)
```

Comment:
The other APIs from KIP-500 is just closed. Maybe this API could follow same pattern?

Reply:
This call needs to be forwarded to the KIP-500 controller.  That is different from the other KIP-500 RPCs which are controller RPCs, and are not expected to be received on the broker at all.

Reply:
@cmccabe Thanks for explanation. It seems the [comment](https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/server/KafkaApis.scala#L221) gets invalid now.
```
Handle requests that should have been sent to the KIP-500 controlle
```

Could you revise the comment as well?

Reply:
The comment is not invalid.  It just doesn't apply to UNREGISTER_BROKER.  It does apply to VOTE, BEGIN_QUORUM_EPOCH, etc. etc.